miqa/core/migrations/0025_globalsettings.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.11 on 2022-01-21 14:23
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_global_project(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor):
+    Project = apps.get_model('core', 'Project')  # noqa: N806
+    GlobalSettings = apps.get_model('core', 'GlobalSettings')  # noqa: N806
+
+    if global_project := Project.objects.filter(name='global').first():
```

Comment:
```suggestion
    if global_project := Project.objects.filter(global_import_export=True).first():
```

Reply:
changed in [b05b85c](https://github.com/OpenImaging/miqa/pull/292/commits/b05b85ca53839899deb48d13b37bca03d683b782)